Ginger/GingerCoreCommon/Repository/SolutionRepository.cs

```diff
@@ -635,6 +667,11 @@ public string CreateRepositoryItemFileName(RepositoryItemBase RI, string contain
             }
         }
 
+        private static void ThrowErrorIfPathLimitExeceeded()
+        {
+            throw new Exception("File path length limit execeeded");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['System.Exception' should not be thrown by user code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4696315)